tests/testthat/test-html-tweak.R

```diff
@@ -162,9 +162,20 @@ test_that("badges can't contain an extra text", {
   )
 })
 
-test_that("badges can be in special div", {
-  expect_equal(
-    badges_extract_text('<p></p><div id="badges"><a href="x"><img src="y"></a></div>'),
-    '<a href="x"><img src="y"></a>'
-  )
+test_that("badges-paragraph à la usethis can be found", {
```

Comment:
Are you sure it's ok to use non-ASCII character in tests?

Reply:
Removed! :fr: :crying_cat_face: